### DISCUSSION TITLE
How to Swipe Element present in WebView.

### DISCUSSION THREAD

**Discussion by akagupta9** [leancodepl/patrol#798] [Q&A]
I want to Swipe the button - Swipe to Upgrade in the below screen. its web view, I can only see one method swipe which needs offsets.
![Uploading Screenshot 2023-01-19 at 8.36.04 PM.png…]()

**Comment by bartekpacia** [comment]
Hi, unfortunately, the screenshot you sent is corrupted. You can edit your post and reupload it.

But I know what feature you have in mind – this is currently not possible using native selectors, although I know it'd be useful.

So the request is to add [dragUntilVisible()](https://pub.dev/documentation/patrol/latest/patrol/PatrolTester/dragUntilVisible.html) but for native selectors. I created an issue for this [here](https://github.com/leancodepl/patrol/issues/799).

**Reply by akagupta9** [reply]
Hi @bartekpacia , I want to drag Icon, swipe to Upgrade. It would be good if we have drag feature based on selector the way we have in flutter.

**Reply by bartekpacia** [reply]
Thanks for providing more info! 

> It would be good if we have drag feature based on selector the way we have in flutter.

I agree it'd be great :)

The way it would work, off the top of my head:
- find coordinates of the `>` button
- swipe right till the end of the screen

This is not possible now unfortunately :(
